sdk/batch/Microsoft.Azure.Batch/CHANGELOG.md

```diff
@@ -1,22 +1,22 @@
-# Microsoft.Azure.Batch release notes
+# 12.1.0-preview.1
```

Comment:
Why is there an entry here for a version that doesn't (and likely will not) exist?

Reply:
The idea is to have an entry for the current version of the package. Then notes can be continuously added till the time of release. If it is empty at release time the build will break.

Reply:
There is validation that this version and the actual version being published match, yes?

So if I say, update my nuspec to a different version (remove preview) but forget it here, that'll be a build failure right?

Reply:
Right. Missing version here will break PR/CI